src/scss/cdb-components/shapes/move.scss

```diff
@@ -128,3 +148,10 @@ Description
     background: $cWhite;
   }
 }
+
+.CDB-Shape-MoveItem.is-grey {
```

Comment:
BTW, move item should be in camel case, not starting with uppercase, seems like we have that issue in several components.